### PR TITLE
Release/1.6.1

### PR DESCRIPTION
**Pull request by DimCitus**
See #761

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,36 @@
+### pg_auto_failover v1.6.1 (July 7, 2021) ###
+
+This release contains monitor schema changes, so the monitor extension gets
+a version bump from 1.5 to 1.6, and this is the first release in the 1.6
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
```
Comment (JelteF) [inline CHANGELOG.md:9]: Should contain something along the lines of:

This means that the new `pg_autoctl drop node` will only complete when the node that is being dropped is still reachable. To drop a node that is unreachable (e.g. machine died), you should now use `pg_autoctl drop node --force`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+* Have pg_autoctl drop node comment wait until the node has been removed. (#748)
+
+#### Fixed
+* Fix/consider timeline when reaching secondary (#695)
+* Install btree_gist when upgrade to >= 1.4, not just 1.4. (#714)
+* Fix a race condition issue around health check updates. (#720)
+* Not all the monitor code got the memo about nodeid being a bigint. (#729)
+* Use sysctl(3) on BSD (#733)
+* Fix transaction begin failure handling (#751)
+* Fix monitor postgres not starting up the first time (#752)
```
Comment (JelteF) [inline CHANGELOG.md:31]: This bug was never present in a release, so I think it can be removed from the changelog.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,36 @@
...
+* Allow building pg_auto_failover with Postgres 14. (#716)
+* Fix Postgres 14 beta2 incompatibility. (#739)
+* Track nodes current timeline on the monitor. (#730)
+* Implement drop-at-a-distance semantics. (#734)
+* Add the reported timeline id to the monitor events table. (#753)
+
+#### Changed
+* Fix how many nodes need to report their LSN to perform a failover. (#707)
+* Allow an old primary node (demoted/catchingup) to join another election. (#727)
+* Have pg_autoctl drop node comment wait until the node has been removed. (#748)
```
Comment (JelteF) [inline CHANGELOG.md:22]:
```suggestion
* Have pg_autoctl drop node wait until the node has been removed. (#748)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,36 @@
...
+This release contains monitor schema changes, so the monitor extension gets
+a version bump from 1.5 to 1.6, and this is the first release in the 1.6
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
+
+#### Added
+* Feature crash recovery before pg rewind (#656)
```
Comment (JelteF) [inline CHANGELOG.md:12]:
```suggestion
* Feature crash recovery before pg_rewind (#656)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,36 @@
...
+* Allow building pg_auto_failover with Postgres 14. (#716)
+* Fix Postgres 14 beta2 incompatibility. (#739)
+* Track nodes current timeline on the monitor. (#730)
```
Comment (JelteF) [inline CHANGELOG.md:15]:
```suggestion
* Track nodes current timeline on the monitor and use it during election. (#730)
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,36 @@
...
+series.
+
+In this release we introduce a new state in the FSM: "dropped". This state
+allows a node to realise it's been dropped from the monitor, and act
+accordingly (mostly, stops cleanly and register it's been dropped).
+
+#### Added
+* Feature crash recovery before pg rewind (#656)
+* Allow building pg_auto_failover with Postgres 14. (#716)
+* Fix Postgres 14 beta2 incompatibility. (#739)
```
Comment (JelteF) [inline CHANGELOG.md:14]: This bug was never present in a released version, so we can skip this from the release notes.

File: CHANGELOG.md (hunk 1 above)
Comment (DimCitus) [reply CHANGELOG.md:9]: Good point, added now.

File: CHANGELOG.md (hunk 2 above)
Comment (DimCitus) [reply CHANGELOG.md]: Yeah, I wasn't sure. Thanks.

File: CHANGELOG.md (hunk 3 above)
Comment (DimCitus) [reply CHANGELOG.md]: Done now, thanks.

File: CHANGELOG.md (hunk 4 above)
Comment (DimCitus) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 5 above)
Comment (DimCitus) [reply CHANGELOG.md]: Normally we would use the wording of the git commit (or PR merge), though in that case it's better to help the users read the notes, so added too.

File: CHANGELOG.md (hunk 6 above)
Comment (DimCitus) [reply CHANGELOG.md]: I wanted to show we've been following along for Postgres 14 ;-) that said, removed.

**Review by JelteF — APPROVED** [review]

**Pull request merged**